docs/content/policy-reference.md

```diff
@@ -347,6 +347,7 @@ complex types.
 | <span class="opa-keep-it-together">``output := format_int(number, base)``</span> | ``output`` is string representation of ``number`` in the given ``base`` | ✅ |
 | <span class="opa-keep-it-together">``output := indexof(string, search)``</span> | ``output`` is the index inside ``string`` where ``search`` first occurs, or -1 if ``search`` does not exist | ✅ |
 | <span class="opa-keep-it-together">``output := lower(string)``</span> | ``output`` is ``string`` after converting to lower case | ✅ |
+| <span class="opa-keep-it-together">``output := strings.repeat(string, times)``</span> | ``output`` is a ``string`` representing ``string`` repeated N ``times`` | SDK dependant |
```

Comment:
Nit: What about `output  is ``string`` repeated ``count`` number of times.` ?

Also `SDK dependant` -> `SDK-dependent`